src/btree/bt_vrfy.c

```diff
@@ -465,11 +533,21 @@ __verify_tree(
           __verify_addr_string(session, ref, vs->tmp1), __wt_page_type_string(page->type),
           page->dsk->write_gen));
 
+    my_stack_level = WT_MIN(vs->depth, WT_ELEMENTS(vs->depth_internal) - 1);
+
     /* Track the shape of the tree. */
     if (F_ISSET(ref, WT_REF_FLAG_INTERNAL))
-        ++vs->depth_internal[WT_MIN(vs->depth, WT_ELEMENTS(vs->depth_internal) - 1)];
+        ++vs->depth_internal[my_stack_level];
     else
-        ++vs->depth_leaf[WT_MIN(vs->depth, WT_ELEMENTS(vs->depth_internal) - 1)];
+        ++vs->depth_leaf[my_stack_level];
+
+    if (vs->dump_tree_shape) {
+        printf("%s  %s%s", __tree_stack(vs),
+          vs->depth <= 1                       ? "ROOT" :
```

Comment:
Root should just be internal?

Reply:
Yes, root is a kind of an internal page, I made it "root" so that it's clearer.

But for the sake of automatic parsers, it's better to make it consistent. I'll remove this special case.

Thanks for noting it!